packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -340,9 +390,39 @@ export class VaTextInput {
     const pattern = this.getPattern();
 
     const currencyWrapper = classnames({
-      'currency-wrapper': currency
+      'currency-wrapper': currency,
+      'prefix-wrapper ': inputPrefix,
+      'suffix-wrapper ': inputSuffix
```

Comment:
There are USWDS classes for prefix and suffix. Are you able to use any of these? You will probably need to import the `usa-input-prefix-suffix` package.

https://designsystem.digital.gov/components/input-prefix-suffix/

Reply:
I did not realize that, thanks for pointing it out. I can push an update for it.